### PR TITLE
Change image host to eu.gcr.io

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Change from gcr.io to eu.gcr.io
- Change from us.gcr.io to eu.gcr.io
We are charged more when transferring our images from us.gcr.io host to our infrastructure in Europe. This PR changes the image host to Europe to benefit from the lower costs

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- [PLATOPS-16](https://airqoteam.atlassian.net/browse/PLATOPS-16?atlOrigin=eyJpIjoiODllY2UyMTc1ODVmNDI1ZDg4ZTNlZjRiNmVhNDUwNGMiLCJwIjoiaiJ9)

**_ARE THERE ANY RELATED PRs?_**
- #1271

[PLATOPS-16]: https://airqoteam.atlassian.net/browse/PLATOPS-16?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: workflow-trigger [hunk 1]
```diff
@@ -1 +1 @@
-2022-12-13T20:00:00Z
+2023-01-26T20:00:00Z
```
Comment (123MwanjeMike) [inline workflow-trigger:1]: Trigger deploy all microservices

File: k8s/fault-detection/prod-fault-detection-api.yaml [hunk 2]
```diff
@@ -62,7 +62,7 @@ spec:
               mountPath: /usr/models
       containers:
         - name: fault-detection-api
-          image: us.gcr.io/airqo-250220/prod-fault-detection-api:latest
+          image: eu.gcr.io/airqo-250220/prod-fault-detection-api:latest
```
Comment (123MwanjeMike) [inline k8s/fault-detection/prod-fault-detection-api.yaml:65]: Change image host

File: k8s/exceedance/prod-airqo-exceedance-job.yaml [hunk 3]
```diff
@@ -24,15 +24,15 @@ spec:
...
-                    values:
-                    - compute
+                - weight: 10
+                  preference:
+                    matchExpressions:
+                      - key: node-type
+                        operator: In
+                        values:
+                          - compute
           containers:
```
Comment (123MwanjeMike) [inline k8s/exceedance/prod-airqo-exceedance-job.yaml:34]: Linting changes made by prettier pre commit hook

File: .github/workflows/deploy-apis-to-production.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@ on:
       - closed
 
 env:
-  REGISTRY_URL: gcr.io
+  REGISTRY_URL: eu.gcr.io
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:11]: Change image host from gcr.io to eu.gcr.io

**Review by Baalmart — APPROVED** [review]
This looks good for me. Thanks @123MwanjeMike

**Review by Baalmart — COMMENTED** [review]
Hi @123MwanjeMike , could we merge this on Friday evening so that we have the weekend (gap) to address any shocks? :)

**Comment by 123MwanjeMike** [comment]
> Hi @123MwanjeMike , could we merge this on Friday evening so that we have the weekend (gap) to address any shocks? :)

This is however already in effect on staging @Baalmart

**Review by NoahNsimbe — APPROVED** [review]
LGTM thanks @123MwanjeMike

**Pull request merged**